.jshintignore

```diff
@@ -0,0 +1,6 @@
+satnogsclient/web/static/js/bootstrap-datetimepicker.js
+satnogsclient/web/static/js/bootstrap.min.js
+satnogsclient/web/static/js/jquery-1.11.3.min.js
+satnogsclient/web/static/js/jquery-3.0.0.min.js
+satnogsclient/web/static/js/js.cookie.js
+satnogsclient/web/static/js/moment.js
```

Comment:
Explicit listing of files should go away and replaced by ignoring a single lib directory.